bundle/manifests/extensions.hive.openshift.io_imageclusterinstalls.yaml

```diff
@@ -185,6 +185,11 @@ spec:
               nodeIP:
                 description: NodeIP is the desired IP for the host
                 type: string
+              ssh_key:
```

Comment:
Underscore?

Reply:
Ah yeah, the bundle wasn't generated with your PR and you set it with the underscore, but I can fix that separately.

ref: https://github.com/openshift/image-based-install-operator/pull/35/files#diff-126f7ef4b1c14a1e0e4386760e8a5398759328cc3fe36a104ee0cafbc6fa932dR65

Reply:
Thanks